sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,217 @@
+
+## Related Work
+
+### [WBTC](https://wbtc.network/assets/wrapped-tokens-whitepaper.pdf)
+
+WBTC is a closed membership system. It is made up of 50+ merchants and custodians with keys to the WBTC multisig contract on Ethereum. WBTC deposits and withdrawals can only be performed by the authorized merchants, and end users purchase WBTC directly from the merchants. Although the merchants manage issuance and redemption, all BTC backing WBTC is held by a single custodian.
+
+### [tBTC v2](https://whitepaper.io/document/691/tbtc-whitepaper)
+
+tBTC is an open membership system, where the BTC is managed by a rotating set of randomly selected nodes which manage a threshold wallet. The system requires that 51-of-100 randomly selected wallet signers must collaborate to produce a proper signature.
+
+### [RBTC](https://rootstock.io/static/a79b27d4889409602174df4710102056/RS-whitepaper.pdf)
+
+rBTC, Rootstock’s (RSK) 2-way peg protocol, called “the Powpeg,” is a closed membership system. Peg operations settle to Bitcoin via merge mining on the RSK side-chain. Instead of collateralizing the system with a new token, peg operators are incentivized by earning a portion of transaction fees. PowPeg operators keep specialized hardware called PowHSMs active and connected to special types of Rootstock full nodes. Since the Bitcoin blockchain and the Rootstock sidechain are not entangled in a single blockchain or in a parent-child relation, peg-in and peg-out transactions require a high number of block confirmations. Peg-ins require 100 Bitcoin blocks, and peg-outs require 4000 Rootstock blocks.
+
+## Activation
+
+sBTC is designed to activate on Stacks Nakamoto as defined in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md). Therefore, this SIP is only meaningful when SIP-021 activates. The sBTC Working Group plans to observe at least 2-4 weeks of network behavior on Stacks Nakamoto to ensure a stable release. After this period, sBTC can be activated on the Stacks network without requiring a separate hard fork.
+
+### Process of Activation
+
+Users can vote to approve this SIP with either their locked/stacked STX or with unlocked/liquid STX, or both. The criteria for the stacker and non-stacker voting is as follows.
+
+#### For Stackers:
+
+In order for this SIP to activate, the following criteria must be met by the set of Stacked STX:
+
+- At least 80 million Stacked STX must vote at all to activate this SIP.
+- Of the Stacked STX that vote, at least 66% of them must vote "yes."
+
+The voting addresses will be:
+
+- **Bitcoin Yes Address:** `3Jq9UT81fnT2t24XjNVY7wijpsSmNSivbK`  
```

Comment:
Can you explain the origins of these addresses?

In prior SIPs, we'd encode a string like "yes-to-3.0" or "no-to-3.0" as base58check strings, so it's unambiguous that these addresses aren't being used to e.g. line the pockets of the SIP authors, or e.g. are being used by some other protocol which would skew or obfuscate the vote.

Reply:
Address in f09f236, thank you!

Reply:
lgtm